<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Informacoes">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>711</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(85, 0, 0);</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>180</y>
        <width>551</width>
        <height>411</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QPushButton" name="btn_cadastrar">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>360</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(85, 0, 0);
font: 12pt &quot;Cantarell&quot;;</string>
       </property>
       <property name="text">
        <string>Cadastrar</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>541</width>
         <height>321</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="Nome">
              <property name="styleSheet">
               <string notr="true">font: 87 10pt &quot;Arial Black&quot;;
color:rgb(255, 255, 255)</string>
              </property>
              <property name="text">
               <string>NOME</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="insert_nome">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="Tel">
              <property name="styleSheet">
               <string notr="true">font: 87 10pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>TELEFONE</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="insert_tel">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="Cpf">
              <property name="styleSheet">
               <string notr="true">font: 87 10pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>CPF</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="insert_cpf">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">font: 87 12pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>SELECIONE O MODELO:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="selCarro">
          <property name="contextMenuPolicy">
           <enum>Qt::NoContextMenu</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">
color: rgb(255, 255, 255);</string>
          </property>
          <item>
           <property name="text">
            <string>Renault Kwid</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Fiat Argo</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Polo TSI</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">font: 87 12pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>SELECIONE O PLANO:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="selplano">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <item>
           <property name="text">
            <string>Plano1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Plano2</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">font: 87 12pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>DATA DE RETIRADA:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="Data">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>0</y>
        <width>191</width>
        <height>171</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/Logo Locar.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>160</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 87 12pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>DADOS CADASTRAIS</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QTableWidget" name="tabela">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>701</width>
        <height>411</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(85, 0, 0);</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
      <column>
       <property name="text">
        <string>NOME</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>CPF</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>TELEFONE</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>CARRO</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PLANO</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>RETIRADA</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="btn_ordenanome">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>430</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>Ordenar por Nome</string>
      </property>
     </widget>
     <widget class="QPushButton" name="excluirCliente">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>430</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>Excluir Cliente</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ocultarCliente">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>430</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Ocultar Cliente</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mostrarOcultos">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>470</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Mostrar clientes ocultos</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ordenar_data">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>470</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>Ordenar por Data</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_lucro">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>520</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>520</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Calcular Lucro Total</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDados_Cadastrais">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionSalvar"/>
    <addaction name="actionCarregar"/>
   </widget>
   <addaction name="menuDados_Cadastrais"/>
  </widget>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
  <action name="actionCarregar">
   <property name="text">
    <string>Carregar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
